cmd/agent/main.go

```diff
@@ -20,12 +20,12 @@ import (
 	"context"
 
 	"github.com/numaproj-labs/numaplane/internal/agent"
-	"github.com/numaproj-labs/numaplane/internal/util/logging"
+	"github.com/numaproj-labs/numaplane/internal/util/logger"
 )
 
 var (
 	// logger is the global logger for the numaplane-agent
-	logger     = logging.NewLogger().Named("numaplane-agent")
+	numaLogger = logger.New(nil, nil).WithName("numaplane-agent")
```

Comment:
As discussed add a new method `NewWithWriter` and keep this one (`New`) without any parameters if we don't need them.